src/Microsoft.SqlTools.ServiceLayer/DisasterRecovery/Contracts/RestoreRequestParams.cs

```diff
@@ -140,6 +141,26 @@ internal IEnumerable<string> SelectedBackupSets
                 SetOptionValue(RestoreOptionsHelper.SelectedBackupSets, value);
             }
         }
+
+        /// <summary>
+        /// The executation mode for the operation. default is execution
+        /// </summary>
+        public TaskExecutionMode TaskExecutionMode { get; set; }
+
+        /// <summary>
+        /// Target Database name
+        /// </summary>
+        public string DatabaseName
+        {
+            get
+            {
+                return TargetDatabaseName;
```

Comment:
why do we have a property that gets & sets another property?  Should we remove this property or the other one?